CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ yarn
 
 After installing package dependencies, go to example app and install it dependencies as well.
 ```shell
-cd example && yarn
+cd example && npm install
```

Comment:
```suggestion
cd example && yarn install
```
I think that we can maintain yarn recommendation

Reply:
Okay revert this change back to original